client/src/templates/Introduction/components/block.tsx

```diff
@@ -193,9 +193,9 @@ export class Block extends Component<BlockProps> {
               }}
             >
               <Caret />
-              <h4 className='course-title'>
+              <div className='course-title'>
                 {`${isExpanded ? collapseText : expandText}`}
-              </h4>
+              </div>
```

Comment:
Some extra text for screen readers is needed here. At the moment each one just says 'expand course for 0/39 collapsed' but there's no information about which course it's referring to or what the 0/39 means and there are several of these on the page.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I've added the course name to the toggle button and also improved the reading of the course stats for screen readers. I'm still not 100% sure I've done the translations correctly on this part though.

Reply:
It definitely sounds much better now!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good